core-api/api_container_service.proto

```diff
@@ -118,6 +121,21 @@ message PortBinding {
   string interface_port = 2;
 }
 
+// ==============================================================================================
+//                                       Get Service Info
+// ==============================================================================================
+message GetServiceInfoArgs {
+  //The service ID to get the info from
+  string service_id = 1;
+}
+
+message GetServiceInfoResponse {
+  // The IP address of the service
+  string ip_addr = 1;
+  // The full path where the API container should execute the suite execution volume on the service container
```

Comment:
1) Let's call this `suite_execution_volume_mount_dirpath` for clarity
2) For this comment, we can clarify to "The full path where the test suite execution volume is mounted on the service" (since it's actually already mounted there at the point of calling `GetServiceInfo`)